### PR TITLE
Fix Paddle version keys gap for supported dtypes

### PR DESCRIPTION
**Pull request by kgmann**

### CODE REVIEW

File: ivy/functional/backends/paddle/__init__.py [hunk 1]
```diff
@@ -137,37 +137,37 @@
...
         ivy.uint32,
         ivy.uint64,
     ),
 }
 
-invalid_int_dtypes = {"2.5.1 and below": (ivy.uint16, ivy.uint32, ivy.uint64)}
-invalid_float_dtypes = {"2.4.2 and below": (ivy.bfloat16,), "2.5.1 and above": ()}
-invalid_uint_dtypes = {"2.5.1 and below": (ivy.uint16, ivy.uint32, ivy.uint64)}
-invalid_complex_dtypes = {"2.5.1 and below": ()}
+invalid_int_dtypes = {"2.4.1 and below": (ivy.uint16, ivy.uint32, ivy.uint64)}
```
Comment (RickSanchezStoic) [inline ivy/functional/backends/paddle/__init__.py:167]: These didn't need changes unless the behavior in version 2.5.1 has changed in which case it needs to be added to the keys here, so if this isn't the case the old keys were correct here.

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/backends/paddle/__init__.py (hunk 1 above)
Comment (kgmann) [reply ivy/functional/backends/paddle/__init__.py]: Sure. The behavior only changed for `invalid_float_dtypes` so I'll revert the changes for the others.

File: ivy/functional/backends/paddle/__init__.py [hunk 2]
```diff
@@ -120,11 +120,11 @@
...
 }
 valid_float_dtypes = {
-    "2.4.2 and below": (ivy.float16, ivy.float32, ivy.float64),
-    "2.5.1 and above": (ivy.bfloat16, ivy.float16, ivy.float32, ivy.float64),
+    "2.4.0 and below": (ivy.float16, ivy.float32, ivy.float64),
+    "2.4.1 and above": (ivy.bfloat16, ivy.float16, ivy.float32, ivy.float64),
 }
-valid_uint_dtypes = {"2.5.1 and below": (ivy.uint8,)}
-valid_complex_dtypes = {"2.5.1 and below": (ivy.complex64, ivy.complex128)}
+valid_uint_dtypes = {"2.4.1 and below": (ivy.uint8,)}
```
Comment (RickSanchezStoic) [inline ivy/functional/backends/paddle/__init__.py:126]: these need to have keys for versions later than 2.4.1

File: ivy/functional/backends/paddle/__init__.py [hunk 3]
```diff
@@ -120,11 +120,11 @@
...
+valid_complex_dtypes = {"2.4.1 and below": (ivy.complex64, ivy.complex128)}
```
Comment (RickSanchezStoic) [inline ivy/functional/backends/paddle/__init__.py:127]: these need to have keys for versions later than 2.4.1

Comment (RickSanchezStoic) [inline ivy/functional/backends/paddle/__init__.py:127]: these need to have keys for versions later than 2.4.1

**Review by RickSanchezStoic — CHANGES_REQUESTED** [review]

**Comment by RickSanchezStoic** [comment]
lgtm! @kgmann thanks!

**Pull request merged**